core/interfaces.go

```diff
@@ -11,9 +11,9 @@ import (
 	"net/http"
 	"time"
 
-	jose "github.com/letsencrypt/boulder/Godeps/_workspace/src/github.com/letsencrypt/go-jose"
+	"github.com/letsencrypt/boulder/Godeps/_workspace/src/github.com/letsencrypt/go-jose"
```

Comment:
okay these are the only things i'm not stoked about because I can't ever remember what package names these import statements create.

did a tool do this change or did you?